index.js

```diff
@@ -176,7 +176,7 @@ Choo.prototype.mount = function mount (selector) {
 }
 
 Choo.prototype.toString = function (location, state) {
-  this.state = xtend({ events: xtend(this._events) }, this.state || {})
+  this.state = xtend(this.state || {}, state || {})
```

Comment:
`this.state` always exists; think we should be able to get away with `xtend(this.state, state || {})`